src/driver/mysql/MysqlDriver.ts

```diff
@@ -1166,7 +1163,7 @@ export class MysqlDriver implements Driver {
     /**
      * Creates an escaped parameter.
      */
-    createParameter(parameterName: string, index: number): string {
+    createParameter(): string {
```

Comment:
This should not be changed - it's the same signature in all drivers. It's also part of the public API, and changing it is a breaking change.